jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -115,15 +115,19 @@
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
               (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
               (send *ri* :wait-interpolation)
-              (setq hcv (send *ri* :get-val '_hard-coded-variables))
+              (setq offset-avoid-bin-top
+                    (gethash :offset-avoid-bin-top (send *ri* :get-val '_hard-coded-variables)))
               (if (and object-length (> 0.20 object-length))
                 (setq entrance-x-list (list -200 -150 -100 -50 0 150))
-                (setq entrance-x-list (list -200 -150 -100 -50 0 150 200)))
+                (progn
+                  (setq entrance-x-list (list -200 -150 -100 -50 0 150 200))
+                  (setq offset-avoid-bin-top 0)
+                  (send *ri* :angle-vector (send *baxter* :rotate-gripper arm 45 :relative nil))))
               (setq avs-picked->place-bin
                     (mapcar
                       #'(lambda (x)
                           (send *ri* :ik->bin-entrance arm target-bin
-                                :offset (float-vector x 0 (gethash :offset-avoid-bin-top hcv))))
+                                :offset (float-vector x 0 offset-avoid-bin-top)))
```

Comment:
No. You change gripper angle at https://github.com/knorth55/jsk_apc/blob/21d34ab98d90d15453aca61027e0909479cad2c8/jsk_2016_01_baxter_apc/euslisp/main-stow.l#L125, but gripper angle becomes 90 degrees in `:ik->bin-entrance` without `gripper-angle` key